src/gui/qt/settings/general_tab.h

```diff
@@ -49,15 +49,19 @@ class GeneralTab : public QFrame {
 private:
   QLabel *defaultGameLabel{new QLabel(this)};
   QLabel *languageLabel{new QLabel(this)};
+#if QT_VERSION >= QT_VERSION_CHECK(6, 5, 0)
   QLabel *themeLabel{new QLabel(this)};
+#endif
   QLabel *updateMasterlistLabel{new QLabel(this)};
   QLabel *checkUpdatesLabel{new QLabel(this)};
   QLabel *loggingLabel{new QLabel(this)};
   QLabel *useNoSortingChangesDialogLabel{new QLabel(this)};
   QLabel *preludeSourceLabel{new QLabel(this)};
   QComboBox *defaultGameComboBox{new QComboBox(this)};
   QComboBox *languageComboBox{new QComboBox(this)};
+#if QT_VERSION >= QT_VERSION_CHECK(6, 5, 0)
```

Comment:
I don't think this is necessary.

Reply:
my idea was not to confuse users with settings which don't do anything, but guarding only `Qt::ColorScheme` is also fine.

Reply:
The theme settings predate the update to 6.5 though, they should work independently.